.circleci/bin/test-airflow

```diff
@@ -54,6 +54,10 @@ AIRFLOW_VERSION=$(docker inspect --format '{{ index .Config.Labels "io.astronome
 export AIRFLOW_VERSION
 echo "Airflow Version: $AIRFLOW_VERSION"
 
+CHART_VERSION=$(helm show chart astronomer/airflow)
+export CHART_VERSION
+echo "Chart Version: $CHART_VERSION"
```

Comment:
This gets everything from `helm show chart`, not just the version. We then can't use it in the `helm install` later.

Reply:
Oh yeah missed it, updated